migrations/networks_config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "test": {
+    "domainID": "1",
+    "erc721Name": "ERC721Test",
+    "erc721Symbol": "ERC721TST",
+    "erc721URI": "metadata.test",
+    "erc20Name": "ERC20Test",
+    "erc20Symbol": "ERC20TST",
+    "erc20LRName": "ERC20Test",
+    "erc20LRSymbol": "ERC20TST",
+    "erc20ResourceID": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "genericResourceID": "0x0000000000000000000000000000000000000000000000000000000000000100",
+    "erc721ResourceID": "0x0000000000000000000000000000000000000000000000000000000000000200",
+    "erc20LockReleaseResourceID": "0x0000000000000000000000000000000000000000000000000000000000000300",
+    "MPCAddress": "0x1c5541A79AcC662ab2D2647F3B141a3B7Cdb2Ae4"
+  },
+  "goerli": {
+    "domainID": "2",
+    "erc721Name": "ERC721Test",
+    "erc721Symbol": "ERC721TST",
+    "erc721URI": "metadata.test",
+    "erc20Name": "ERC20Test",
+    "erc20Symbol": "ERC20TST",
+    "erc20LRName": "ERC20Test",
+    "erc20LRSymbol": "ERC20TST",
+    "erc20ResourceID": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "genericResourceID": "0x0000000000000000000000000000000000000000000000000000000000000100",
+    "erc721ResourceID": "0x0000000000000000000000000000000000000000000000000000000000000200",
+    "erc20LockReleaseResourceID": "0x0000000000000000000000000000000000000000000000000000000000000300",
+    "MPCAddress": "0x1c5541A79AcC662ab2D2647F3B141a3B7Cdb2Ae4"
+  },
+  "mumbai": {
+    "domainID": "3",
+    "erc721Name": "ERC721Test",
+    "erc721Symbol": "ERC721TST",
+    "erc721URI": "metadata.test",
+    "erc20Name": "ERC20Test",
+    "erc20Symbol": "ERC20TST",
+    "erc20LRName": "ERC20Test",
+    "erc20LRSymbol": "ERC20TST",
```

Comment:
what  *LR* means? Why symbols are the same?

Reply:
ah ok lock release, but lets make symbols different anyway

Reply:
Yeah, missed it :sweat_smile: